app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/MainScaffold.kt

```diff
@@ -0,0 +1,104 @@
+/*
+ * Infomaniak SwissTransfer - Android
+ * Copyright (C) 2024 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.infomaniak.swisstransfer.ui.screen.main
+
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.material3.adaptive.currentWindowAdaptiveInfo
+import androidx.compose.material3.adaptive.navigationsuite.NavigationSuiteScaffold
+import androidx.compose.material3.adaptive.navigationsuite.NavigationSuiteScaffoldDefaults
+import androidx.compose.material3.adaptive.navigationsuite.NavigationSuiteType
+import androidx.compose.runtime.*
+import androidx.compose.ui.res.stringResource
+import androidx.navigation.NavGraph.Companion.findStartDestination
+import androidx.navigation.NavHostController
+import com.infomaniak.swisstransfer.ui.navigation.MainNavigation
+import com.infomaniak.swisstransfer.ui.navigation.NavigationItem
+
+@Composable
+fun MainScaffold(
+    navController: NavHostController,
+    currentDestination: MainNavigation,
+    content: @Composable () -> Unit = {},
+) {
+    val adaptiveInfo by rememberUpdatedState(currentWindowAdaptiveInfo())
+
+    val showBottomBar by remember(currentDestination) {
+        derivedStateOf {
+            NavigationItem.entries.any { it.destination == currentDestination }
+        }
+    }
+
+    val navType by remember(showBottomBar, adaptiveInfo) {
+        derivedStateOf {
+            NavigationSuiteScaffoldDefaults.calculateFromAdaptiveInfo(adaptiveInfo)
+        }
+    }
+
+    MainScaffold(
+        navType = navType,
+        currentDestination = currentDestination,
+        navController = navController,
+        content = content,
```

Comment:
Do we really need to specify the parameter name here ? Since the name of the variable is the same as the function parameter.

Reply:
This code is outdated